fenix/app/src/main/res/drawable/ic_download_circle_spinner.xml

```diff
@@ -0,0 +1,16 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
When it comes to adding icons, we should pull the resource from https://github.com/FirefoxUX/acorn-icons/tree/main/icons/mobile. You will see the icons that are available in our iconography in this [figma](https://www.figma.com/file/Kv7wWr8vH6AQrGhu9vmjdE/Mobile-Icons?node-id=221%3A1223&mode=dev). These icons will have their paths optimized and formatted in acorn-icons, which we should treat as our source of truth.

In this case, we didn't need to import this because it should be the equivalent of [mozac_ic_sync_24.xml](https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/ui/icons/src/main/res/drawable/mozac_ic_sync_24.xml). I am in the middle of updating our existing icons to use the latest paths in acorn-icons as part of [Bug 1842389](https://bugzilla.mozilla.org/show_bug.cgi?id=1842389). For the time being, I think we can remove this newly added icon and just use `mozac_ic_sync_24.xml`

Reply:
Done